samples/client/petstore/csharp/OpenAPIClient/docs/FakeApi.md

```diff
@@ -645,14 +645,14 @@ namespace Example
             var _double = 1.2D;  // double | None
             var patternWithoutDelimiter = patternWithoutDelimiter_example;  // string | None
             var _byte = BYTE_ARRAY_DATA_HERE;  // byte[] | None
-            var integer = 56;  // int | None (optional) 
-            var int32 = 56;  // int | None (optional) 
-            var int64 = 789;  // long | None (optional) 
-            var _float = 3.4F;  // float | None (optional) 
+            var integer = 56;  // int? | None (optional) 
+            var int32 = 56;  // int? | None (optional) 
+            var int64 = 789;  // long? | None (optional) 
+            var _float = 3.4F;  // float? | None (optional) 
```

Comment:
These are not defined as nullable in the spec but with this PR these become nullable: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml#L709

@gbrown-ce can you please take a look when you've time?

Reply:
@wing328 Unless I'm misunderstanding something, they should be nullable in the generated code. They aren't marked as `required` therefore they're optional, and thus nullable in C#

Reply:
@gbrown-ce you're right. I think I got confused with something else.